x-pack/plugins/index_management/public/application/components/mappings_editor/components/document_fields/fields/create_field/semantic_text/use_semantic_text.ts

```diff
@@ -64,7 +64,22 @@ export function useSemanticText(props: UseSemanticTextProps) {
     }
   }, [form, inferenceId, inferenceToModelIdMap]);
 
-  const handleSemanticText = (data: Field) => {
+  const isModelDownloaded = async (modelId: string) => {
+    try {
+      const response = await ml?.mlApi?.trainedModels.getTrainedModels(modelId, {
+        include: 'definition_status',
+      });
+      // @ts-ignore
+      return !!response[0]?.fully_defined;
```

Comment:
For this TS error, could you please open an issue to fix the wrong/missing type and use `ts-expect` with the link to the issue instead of `ts-ignore`?

Reply:
In order to resolve @ts-ignore issue, I specified the type of response now. So, linter is good for now.